test/integration/spec/localparticipant.js

```diff
@@ -354,7 +354,15 @@ const isSafari = guess === 'safari';
             thoseTracks.forEach(thatTrack => assert.equal(thatTrack.name, thisLocalTrackPublication.trackName));
           });
 
-          if (kind !== 'data') {
+          if (kind === 'data') {
+            it('should transmit any data sent through the LocalDataTrack to the Room to each RemoteDataTrack', async () => {
+              const thoseTracks = thoseTracksMap[event];
+              const thoseTracksReceivedData = thoseTracks.map(track => new Promise(resolve => track.once('message', resolve)));
+              thisTrack.send('foo');
```

Comment:
Actually, just remembered—this won't be a reliable test. We need to do something like

```js
interval = setInterval(() => {
  thisTrack.send('foo');
}, 100);
```

and then

1. Check that at least 1 message gets through
2. `clearInterval` in `after`/`afterEach`